boards/BMS/Src/App/states/App_ChargeState.c

```diff
@@ -25,16 +27,18 @@ static void ChargeStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
     if (App_AllStatesRunOnTick100Hz(state_machine))
     {
-        struct BmsWorld *         world       = App_SharedStateMachine_GetWorld(state_machine);
-        struct BmsCanTxInterface *can_tx      = App_BmsWorld_GetCanTx(world);
-        struct Charger *          charger     = App_BmsWorld_GetCharger(world);
-        struct Accumulator *      accumulator = App_BmsWorld_GetAccumulator(world);
-        struct Airs *             airs        = App_BmsWorld_GetAirs(world);
+        struct BmsWorld *            world   = App_SharedStateMachine_GetWorld(state_machine);
+        struct BmsCanTxInterface *   can_tx  = App_BmsWorld_GetCanTx(world);
+        struct BmsCanRxInterface *   can_rx  = App_BmsWorld_GetCanRx(world);
+        struct Charger *             charger = App_BmsWorld_GetCharger(world);
+        struct Airs *                airs    = App_BmsWorld_GetAirs(world);
+        struct TractiveSystem *const ts      = App_BmsWorld_GetTractiveSystem(world);
 
         static uint16_t ignore_chgr_fault_counter      = 0U;
         const bool      is_charger_disconnected        = !App_Charger_IsConnected(charger);
         bool            has_charger_faulted            = false;
         bool            has_external_shutdown_occurred = !App_Airs_IsAirNegativeClosed(airs);
+        bool            charge_over_can                = App_CanRx_BMS_CHARGER_GetSignal_IS_CHARGING_ENABLED(can_rx);
```

Comment:
```suggestion
        bool            charging_enabled                = App_CanRx_BMS_CHARGER_GetSignal_IS_CHARGING_ENABLED(can_rx);
```
to stick with our naming convention